onnxruntime/python/tools/kernel_explorer/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+project(kernel_explorer)
+cmake_minimum_required(VERSION 3.21)
+
+include(../../../../cmake/external/pybind11.cmake)
+
+# Memory error with Release build (-o3 -DNDEBUG), AMD is currently investigating
+set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_CXX_COMPILER /opt/rocm/bin/hipcc)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fpic")
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hip /opt/rocm)
+find_package(hip)
+find_package(PythonLibs 3.8 EXACT REQUIRED)
+
+include_directories(${PYTHON_INCLUDE_DIRS})
+include_directories(${pybind11_INCLUDE_DIRS})
+include_directories(.)
+
+add_library(timer timer.cpp)
+add_library(vector_add kernels/vector_add.cpp)
```

Comment:
how will we add future kernels? keep update cmake file here?
if we want flexibility, you can consider make the path to kernel folder configurable.

Reply:
Use glob and wildcard for now. Yes, adding configurable path would be nice, will address that in future PR when we need to reference other kernel directories.